app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,9 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
+    <color name="background_btn">#66BB6A</color>
+    <color name="background_layout">#000000</color>

```

Comment:
#000000 is repeated in three different colors.. make the color_black with that value and the background_layout, textcolor and bordercolor use @color/color_black

Reply:
pls if the color already exists do not create it again

Reply:
mmm. I agree with Sarti. its ok to have 3 same colors with different name. 

<color name="color_black">#000000</color>
<color name="background_layout">@color/color_black</color>
<color name="textcolor">@color/color_black</color>
<color name="bordercolor">@color/color_black</color>

Reply:
Done!!